src/adapters/gumgum.js

```diff
@@ -10,8 +10,16 @@ const GumgumAdapter = function GumgumAdapter() {
 
   const bidEndpoint = `https://g2.gumgum.com/hbid/imp`;
 
-  const WINDOW = global.top;
-  const SCREEN = WINDOW.screen;
+  let WINDOW;
```

Comment:
Well we normally reserve caps for constants, anything else should just be camelCase. Also as an FYI we have some helper functions that safely attempt tot access window.top.* variables that might be helpful: https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L193